### PR TITLE
Reee mgame scores indicators

### PR DESCRIPTION
**Pull request by HauHe**
Dear @willu47,
would you be so kind as to have a look on this skeleton of a script and give me feedback if it looks fine?
Thanks a lot for your time!
Best,
Hauke

### CODE REVIEW

File: OSeMBEtoREEEMgame/REEEMgame_scores_indicators.py [hunk 1]
```diff
@@ -6,56 +6,93 @@
...
 import pandas as pd
-
-#%% Import function to import all data needed
-def ImportData():
+import numpy as np
+#%% Import function to import data from REEEMdb
+def import_reeemdb():
     rawData = pd.DataFrame()
     return rawData
+#%% Import from excel file
```
Comment (willu47) [inline OSeMBEtoREEEMgame/REEEMgame_scores_indicators.py:14]: Standard comments for Python are within a docstring like so:
```python
def my_function(argument):
    """Here's a summary line

    Leave a space above, then add more details here.

    Arguments
    ---------
    argument : str
        Add a list of arguments with the type (str, int, float etc.) and then an indented description below it.
    """

**Review by willu47 — COMMENTED** [review]
Hi @HauHe. This is looking good. I think you could add more into the `main` function to show how you'll add columns to the Pandas DataFrame containing the indicators, calling the functions.

Otherwise, this seems like a nice structure.

If you like, I can show you how to set this up so that you can write tests for each of the functions. Perhaps implement a few of them first, and then we can try writing a test together and you can see which approach you prefer.

**Review by HauHe — COMMENTED** [review]
Thanks a lot for the comments @willu47 !
Will aim to keep them in mind!
Could you approve the pull request? GitHub doesn't allow me since I created it...

**Comment by HauHe** [comment]
Ok After writing a review comment I was allowed to merge it :)

**Pull request merged**